docs/authzed/pricing.md

```diff
@@ -0,0 +1,93 @@
+# Pricing on Authzed
+
+[Authzed], the managed permissions database service, has a **usage** based pricing model.
+
+Customers are charged based on two dimensions: the number of [relationships] stored in Authzed, and the number of dispatched operations made when performing [API] calls.
+
+[relationships]: /reference/glossary#relationship
+[API]: /reference/api
+[Authzed]: https://app.authzed.com
+
+| Dimension | Price Per 1 miliion |
+| --------- | ------------------- |
+| Relationships | $1 per 1 million |
+| Dispatched Operations | $1 per 1 million |
+
+## Do you have a free tier?
+
+Yes. Any amount less than 1 million relationships or dispatched operations is considered a charge of `$0`.
```

Comment:
Maybe this should mention dev systems and not just minimal usage of production systems

Reply:
We should probably include that you do have to enter a credit card first for production systems.